### ISSUE TITLE
is brave still a good and safe browser?

### ISSUE THREAD

**Issue by RojjaCebolla** [Tobi823/ffupdater#114]
i keep hearing that brave is increasing its ties to cryptocurrency and (currently opt-in) advertising.  just wanted to check whether you still find brave to be legit and privacy-focused.

**Comment by Tobi823** [comment]
>  just wanted to check whether you still find brave to be legit and privacy-focused.

Personally I only use Mozilla-based browser because I don't want to support the monopoly of the Chromium render engine. On the desktop I've used Firefox for a long time but switch recently to LibreWolf because even Mozilla is not perfect.
On my smartphone I still use Firefox because I don't browse the web that often on it.

In a perfect world, I would only support Firefox and its derivatives. But good people are working on many other projects like UngoogledChromium. Moreover many people want different features. For example the maintainer of the Custom-Android-Rom GrapheneOS recommends the browser Vandium (based on Chromium) because it is more secure. Other people like the advanced features of Vivaldi.

I personally would not use Brave on my smartphone. (But other Chromium-based browser like UngoogledChromium are handy if one website does not in Firefox).

I think once I commit to support a browser there must be a very good reason to remove it. For example, I have to inform the users that their browser is not longer supported. 
It is very likely that they will continue to use the browser and they switch to a less safer method of updating it (for example downloading the update unregulary from GitHub.) 

> i keep hearing that brave is increasing its ties to cryptocurrency and (currently opt-in) advertising

I was aware that Brave was trying to introduce a new way of advertising that could solve the current problems of tracking, etc. (As far as I know it is still unpopular)

But I was not ware that the Brave is increasing its ties to cryptocurrency. According to one source the Basic Attention Token BAT is based on Ethereum :( Etherum wants to switch to "Proof of Stake" but I think they are still using "Proof of Work".

TL/DR:

You are right - the privacy level of Brave is not comparable with other browsers like Firefox. But I think that removing is not a real solution. I will:
- Update the F-Droid page and distinguish between Mozilla, good privacy and "better than Google Chrome" browser
- I will update the displayed warning for Brave in the app
- Update the UI in the app to inform existing user about the drawbacks of Brave

**Comment by Tobi823** [comment]
it's done now and will be part of the next release
